### PR TITLE
Add support for r2

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: diaphora.py [hunk 1]
```diff
@@ -598,7 +598,11 @@ def save_function(self, props):
         for bb in bb_relations[key]:
           bb = str(bb)
           key = str(key)
-          cur_execute(sql, (bb_ids[key], bb_ids[bb]))
+          try:
```
Comment (joxeankoret) [inline diaphora.py]: This is wrong. We should not be ignoring errors because there is no relationship found by Radare.

**Comment by joxeankoret** [comment]
I will only accept the PR under the next 2 points:

1. The bug in diaphora_r2.py or Radare2 I mention in the review is fixed.
2. Radare's people take the responsibility to maintain it.

Also, do not send me directories and files that are exclusive of Radare that reside in the main directory. This is not Radare. Create a directory like contrib/r2/.

**Comment by joxeankoret** [comment]
I still believe it's wrong to continue after errors, but is OK for now due to implications of the bug.

**Comment by radare** [comment]
🦑

**Pull request merged**